build.gradle

```diff
@@ -206,7 +206,9 @@ opensearchplugin {
 loggerUsageCheck.enabled = false
 
 // No need to validate pom, as we do not upload to maven/sonatype
```

Comment:
Hmm, this comment doesn't seem correct as when I look in maven I see the poms https://aws.oss.sonatype.org/content/repositories/snapshots/org/opensearch/plugin/opensearch-security/3.0.0.0-SNAPSHOT/

Reply:
This was an area I was unsure of and finally figured out how to disable all the validate pom tasks. I am currently generating SNAPSHOT jars before and after this change and am verifying that there is no change in contents.